### DISCUSSION TITLE
can we show 2 days in a resourceTimelineDay view

### DISCUSSION THREAD

**Discussion by SweAdeel** [vkurko/calendar#374] [Q&A]
can we show 2 days in a resourceTimelineDay view

please respond

**Comment by vkurko** [comment]
You can try setting [duration](https://github.com/vkurko/calendar?tab=readme-ov-file#duration) option:

```js
duration: {days: 2}
```

**Reply by SweAdeel** [reply]
how to make like
slotmintime : 14:00
slotmaxtime : 02:00 (which will be nextday)

**Reply by SweAdeel** [reply]
like how to put this in duration
duration: { hours: 2, minutes: 30 }

**Reply by vkurko** [reply]
Sorry, сan you explain in more detail what exactly you are having trouble with?
